### PR TITLE
Update requirement sets for Office.Actions

### PR DESCRIPTION
**Pull request by davidchesnut**
Please fill in this template.

- [x ] Use a meaningful title for the pull request. Include the name of the package modified.
- [x ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x ] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x ] Provide a URL to documentation or source code which provides context for the suggested changes: https://docs.microsoft.com/en-us/javascript/api/office/office.actions?view=common-js-preview
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Review by Rick-Kirkham — APPROVED** [review]

File: types/office-js-preview/index.d.ts [hunk 1]
```diff
@@ -583,7 +588,12 @@ declare namespace Office {
          *
          * @remarks
          *
-         * **Requirement set**: {@link https://docs.microsoft.com/javascript/api/requirement-sets/common/keyboard-shortcuts-requirement-sets | KeyboardShortcuts 1.1}
+     * **Requirement sets**:
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts:591]:
```suggestion
         * **Requirement sets**:
```

File: types/office-js/index.d.ts [hunk 2]
```diff
@@ -538,8 +538,6 @@ declare namespace Office {
     /**
      * Manages actions and keyboard shortcuts.
      *
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js/index.d.ts:540]:
```suggestion
```

File: types/office-js-preview/index.d.ts [hunk 3]
```diff
@@ -538,26 +538,26 @@ declare namespace Office {
     /**
      * Manages actions and keyboard shortcuts.
      *
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts:540]:
```suggestion
```

**Review by ElizabethSamuel-MSFT — COMMENTED** [review]
a few nits...

**Pull request merged**